api-mock/mocks/SessionMock.ts

```diff
@@ -0,0 +1,196 @@
+import Session from "supertokens-node/recipe/session";
+import { queryAPI } from "../fetcher";
+import { minify } from "../utils";
+import SuperTokens from "supertokens-node";
+import { RecipeInterface } from "supertokens-node/lib/build/recipe/session/types";
+import SessionClass from "supertokens-node/lib/build/recipe/session/sessionClass";
+
+function serializeSession(session) {
```

Comment:
I think this is actually deserialization, right?
```suggestion
function deserializeSession(session) {
```